lib/zendesk_apps_support/validations/requirements.rb

```diff
@@ -7,7 +7,7 @@ module Requirements
       MAX_CUSTOM_OBJECTS_REQUIREMENTS = 50
 
       class << self
-        def call(package)
+        def call(package, opts = {})
```

Comment:
Could we make the new arg explicit instead of a generic option (e.g. `validate_custom_objects_v2: false`)? That would make it easier to remove later since this should be a temporary change.


```suggestion
        def call(package, validate_custom_objects_v2: false)
```